translations/en-us.yml

```diff
@@ -1612,6 +1612,14 @@ osf-components:
             success: 'Contributor order updated.'
         noEducation: 'No education history to show'
         noEmployment: 'No employment history to show'
+        removeContributor:
+            confirmRemove:
+                title: 'Remove contributor'
+                body: 'Are you sure you want to remove {userName}?'
+                button: 'Remove contributor'
+            success: 'You have successfully removed this contributor.'
```

Comment:
tiny request here, I would also use the contributor name here for the success toast as well